jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryEndpointRequestProperties.java

```diff
@@ -21,14 +21,19 @@
 
   private final String serverUrl;
   private final String imageName;
+  private final String[] dependentImageNames;
 
   /**
    * @param serverUrl the server URL for the registry (for example, {@code gcr.io})
    * @param imageName the image/repository name (also known as, namespace)
+   * @param dependentImageNames any additional images required for access; may be {@code null} or
```

Comment:
Is `required for access` the right explanation here? Maybe it's more like `any additional images that may provide blobs to mount into the {@code imageName} repository`?